operator-common/src/main/java/io/strimzi/operator/common/metrics/MetricsUtils.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.metrics;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.strimzi.operator.common.config.ConfigParameter;
+import io.strimzi.operator.common.model.Labels;
+
+import java.util.Set;
+
+/**
+ * Utility class for metrics
+ */
+public class MetricsUtils {
+    /**
+     * Private constructor to prevent instantiation
+     */
+    private MetricsUtils() {
+        // Intentionally left blank
+    }
+
+    /**
+     * Tag representing any namespace
+     */
+    protected static final Tag TAG_ANY_NAMESPACE = Tag.of("namespace", ConfigParameter.ANY_NAMESPACE);
```

Comment:
Does that tag really exist for any metrics? I do not see it for any metric when watching all namespaces.